src/login/logind-button.h

```diff
@@ -5,6 +5,16 @@ typedef struct Button Button;
 
 #include "logind.h"
 
+typedef enum
+{
+        MOD_NONE = 0,
+        MOD_SHIFT = 1,
+        MOD_CTRL = 1 << 1,
+        MOD_ALT = 1 << 2,
+        MOD_META = 1 << 3,
```

Comment:
please align these, as we usually do. and define shift the same way as the others, i.e. as `1 << 0`